src/oneid/oneid-lambda-assertion/index.py

```diff
@@ -12,6 +13,16 @@
 s3 = boto3.resource("s3")
 bucket_name = os.environ['S3_BUCKET']
 
+def decode_base64_content(content_body):
+    
+ samlreq_base64 = content_body['SAMLRequest']['S']
```

Comment:
I don't know!  Does it make any sense to check any encoding error? 
Would it be possible that the saml request or response contain characters that are not in the utf-8 range (cc @sebbalex )